AmazonChimeSDKDemo/AmazonChimeSDKDemo/AutomaticLanguageDetectionViewController.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  AutomaticLanguageDetectionViewController.swift
+//  AmazonChimeSDKDemo
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AmazonChimeSDK
+import UIKit
+
+class AutomaticLanguageDetectionViewController: UIViewController, UITextFieldDelegate {
+    
+    @IBOutlet var languageDetectionTableView: UITableView!
+    @IBOutlet var preferredLanguagesTextField: UITextField!
+    var preferredLanguagesPickerView = UIPickerView()
+    public var cancellationHandler: ((Int?) -> Void)?
+    public var selectedLanguageOptionsHandler: ((String?) -> Void)?
+    public var preferredLanguageOptionHandler: ((String?) -> Void)?
+    public var languages: [String] = []
+    public var languagesDict: [String: String] = [:]
+    
+    var preferredLanguages = ["-- Optional --"]
+    var selectedLanguages = " "
```

Comment:
Name reason for `" "` instead of `""`. 

Rename as `selectedLanguageOptions` or `languageOptionsSelected` based on the convention.

Also, do we need this variable? I can only see it's use in [code stub](https://github.com/aws/amazon-chime-sdk-ios/pull/411/files#diff-d15bd327f4100805932691ed6dd593ee26f0f0c1f8cd9df25656d03d6739a563R48-R56)

Reply:
will update name in next commit. `selectedLanguageOptionsHandler?(selectedLanguageOptions)` passes this value to back to `liveTranscriptionViewController` to be used in streamParams.